extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js

```diff
@@ -61,28 +68,46 @@ $(function () {
 
 				$ul.sortable('refresh');
 			},
-			checkInputs = function (elements, checkEmpty, required) {
-				var names = [];
+			/**
+			 * Validate input elements
+			 *
+			 * @param elements
+			 * @param options array consists of ['checkEmpty', 'required', 'limit']
+			 */
+			checkInputs = function (elements, options) {
+				var names = [],
+					optionCheckEmpty, optionRequired, optionLimit;
+
+				optionCheckEmpty = typeof options !== 'undefined' ? (options.indexOf('checkEmpty') !== -1) : false;
+				optionRequired = typeof options !== 'undefined' ? (options.indexOf('required') !== -1) : false;
+				optionLimit = typeof options !== 'undefined' ? (options.indexOf('limit') !== -1) : false;
```

Comment:
I would add a check for "is array" instead of 'undefined' if you're expecting an array (as said in comment)